09-ping.exs

```diff
@@ -21,16 +21,20 @@ defmodule Ping do
   Ping a single IP address and return true if there is a response."
   """
   def ping(ip) do
-    result = System.cmd(ping_cmd(ip))
-    not Regex.match?(~r/100(\.0)?% packet loss/, result)
+    # return code should be handled somehow with pattern matching
+    {cmd_output, _} = System.cmd("ping", ping_args(ip))
+    valid_output = "1 packets transmitted, 1 packets received, 0.0% packet loss"
+    String.contains?(cmd_output, valid_output)
```

Comment:
Why remove the regex approach? Seems it would be more platform independent and less reliant on that one output format.